### PR TITLE
[11.0] [MIG] project task add very high

### PR DESCRIPTION
**Pull request by emagdalenaC2i**
* Update README.rst
* Update __init__.py
* Update __manifest__.py
* Update hooks.py
* Update project_task.py

### CODE REVIEW

**Comment by dreispt** [comment]
Commit history is missing.

**Comment by emagdalenaC2i** [comment]
I know, I will fix it in a couple of days.

File: project_task_add_very_high/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (astirpe) [inline project_task_add_very_high/__manifest__.py:1]: Why did you remove the Copyright from the header?

Comment (emagdalenaC2i) [reply project_task_add_very_high/__manifest__.py:1]: I believed that the Copyright should be only in the "author" tag
This is one of the small changes I will do in a few days.

**Comment by astirpe** [comment]
I think this one can be closed

**Comment by emagdalenaC2i** [comment]
Merged https://github.com/OCA/project/pull/343
